src/docs/go-grpc.yaml

```diff
@@ -0,0 +1,19 @@
+slug: go-grpc
+name: Go gRPC
+written: September 14, 2021
+updated: September 14, 2021
+protocol: gRPC
+port: 443
+# paths:
+#   serverCert: /etc/ssl/server.crt
+#   serverKey: /etc/ssl/server.key
+#   rootCert: /etc/ssl/root_ca.crt
+# chart: ingress-nginx
```

Comment:
I think we can just delete the `chart` line.. not even sure 100% how we'll use it.

Reply:
I left all those things commented because I wasn't sure how they were used it.